ivy_tests/test_ivy/test_functional/test_core/test_linalg.py

```diff
@@ -1101,15 +1102,14 @@ def test_matrix_norm(
         keepdims=kd,
     )
 
-
+    
 # matrix_rank
 @given(
     dtype_x=helpers.dtype_and_values(
         available_dtypes=ivy_np.valid_float_dtypes[1:],
-        min_num_dims=3,
-        max_num_dims=3,
-        min_dim_size=2,
-        max_dim_size=3,
+        min_num_dims=2,
+        min_value=-10000000000.0,
```

Comment:
could you write these numbers in `1e<x>` format?

Reply:
Yeah no worries!